### PR TITLE
Update MX

### PR DESCRIPTION
**Pull request by irabinovitch**
This commit preps the changes for the cutover from ActUSA mail servers to Google Apps.
Please do not merge/push until Sunday at noon.

### CODE REVIEW

File: zones/linuxfests/linuxfests.org.zone [hunk 1]
```diff
@@ -2,7 +2,11 @@ $ORIGIN linuxfests.org.
 @	900	IN	SOA	ns-1064.awsdns-05.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400
 @	3600	IN	A	208.83.101.227
 @	3600	IN	AAAA	2607:ff38:2:6::e3
-@	3600	IN	MX	10 mail.linuxfests.org.
+@	3600	IN	MX	1   ASPMX.L.GOOGLE.COM.
```
Comment (jhoblitt) [inline zones/linuxfests/linuxfests.org.zone:5]: is it a requirement to have google at a weight of 1?  Having no lower (higher) weight available will make a migration [away from google] a little bit harder.

**Comment by irabinovitch** [comment]
@jhoblitt  see google's docs:
https://support.google.com/a/answer/33915?hl=en

**Comment by jhoblitt** [comment]
My interpretation of "You may not be able to enter the priority value exactly as it appears in the table below" is that the weights can be anything as long as the ranking order is preserved.

**Comment by jhoblitt** [comment]
Google itself doesn't use a weight of `1`:

```
dig google.com -t mx +short
20 alt1.aspmx.l.google.com.
10 aspmx.l.google.com.
40 alt3.aspmx.l.google.com.
50 alt4.aspmx.l.google.com.
30 alt2.aspmx.l.google.com.
```

**Comment by irabinovitch** [comment]
Feel free to adjust.

**Comment by irabinovitch** [comment]
merging based on discussions in #infra with @jhoblitt

**Pull request merged**